### PR TITLE
Add NodeJS Name Service

### LINKED ISSUE #43: Add node name service
This service returns a random name. It depends on the year service

### PR DESCRIPTION
**Pull request by JamieDanielson**
Closes #43 and #45 

* Adds name service
* Uses name service in frontend (replaces placeholder)
* Adds extra instrumentation for detailed spans
* Updates Tilt file and Docker file

### CODE REVIEW

**Review by vreynolds — APPROVED** [review]

File: node/name-service/main.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+'use strict';
+const beeline = require('honeycomb-beeline')({
```
Comment (vreynolds) [inline node/name-service/main.js:2]: https://docs.honeycomb.io/getting-data-in/javascript/beeline-nodejs/#interoperability-with-opentelemetry

Comment (vreynolds) [reply node/name-service/main.js:2]: Need to add w3c interop, so the beeline plays nice with OTEL (trace interop)

**Review by vreynolds — CHANGES_REQUESTED** [review]
missing trace interop

**Review by vreynolds — APPROVED** [review]

**Comment by JamieDanielson** [comment]
Will create new PR to handle the distributed traces across services with mixed usage of beeline and Otel.

**Pull request merged**